Sources/Buildkite/Resources/TestAnalytics/TestAnalyticsUpload.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  TestAnalyticsUpload.swift
+//  Buildkite
+//
+//  Created by Aaron Sky on 6/5/22.
+//  Copyright © 2022 Aaron Sky. All rights reserved.
+//
+
+import Foundation
+
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
+
+public enum TestAnalytics {
+    /// Resources for requesting information about your organization's Buildkite agents.
+    public enum Resources {}
+}
+
+extension TestAnalytics.Resources {
+    /// Get metrics about agents active with the current organization.
+    public struct Upload: Resource {
+        public typealias Content = String
```

Comment:
I can't land this line until I've fully tested the response, which almost certainly does not respond with this type.